app/views/spree/admin/overview/single_enterprise_dashboard.html.haml

```diff
@@ -40,11 +40,18 @@
         %span.icon-user
         = t "your_profil_live"
     .list
-      %a.button.bottom{href: main_app.enterprise_shop_url(@enterprise), target: '_blank'}
-        = t "see"
-        = @enterprise.name
-        = t "live"
-        %span.icon-arrow-right
+      - if @enterprise.sells == "none"
+        %a.button.bottom{href: main_app.producers_url, target: '_blank'}
```

Comment:
This may not work in all cases. A producer is characterised by the `is_primary_producer` flag. But an enterprise can also not be a producer and not be a shop. In that case it's just a profile, just showing contact details.

Reply:
Thanks for the hint. I created the helper based with `is_distributor`